docscan/sandbox/request/task/document_text_data_extraction_task.go

```diff
@@ -19,6 +19,7 @@ type DocumentTextDataExtractionTaskBuilder struct {
 	documentIDPhoto *documentIDPhoto
 	detectedCountry string
 	recommendation  *TextDataExtractionRecommendation
+	manualCheck     string
```

Comment:
Do we need this line? 

Please can you make sure you install the pre-commit hook for every repo. 
Will be `pre-commit install` from the root, and `pre-commit run --all-files` to run. Running that from this branch gives me:
> docscan/sandbox/request/task/document_text_data_extraction_task.go:22:2: `manualCheck` is unused (structcheck)
        manualCheck     string